echopype/utils/misc.py

```diff
@@ -11,3 +14,43 @@ def camelcase2snakecase(camel_case_str):
             camel_case_str = camel_case_str[:i] + "_" + camel_case_str[i:]
 
     return camel_case_str.lower()
+
+
+def depth_from_pressure(pressure, latitude=30, atm_pres_surf=0):
+    """
+    Convert pressure to depth using UNESCO equation.
+    Parameters
+    ----------
+    pressure : array_like
+        Pressure in dbar
+    latitude : float
+        Latitude in degrees
+    atm_pres_surf : float
+        Atmospheric pressure at the surface in dbar
+    Returns
+    -------
+    depth : array_like
+        Depth in meters
+    """
+    # Constants
```

Comment:
For consistency with UNESCO 1983, please use the names for constants used there. Your k1 - k4 should be mapped to c4 to c1. For your k5 - k7, UNESCO 1983 doesn't define constant names. How about k1 - k4?

Reply:
I also suggest using the signs for constants c1 - c4 from UNESCO. Then, the terms in the `depth_w_g` will all be additions.

Reply:
FYI I double checked the values of all those constants, and they're correct. The signs for some of them will change -- see my comment, above.

Reply:
Updated!